brayns/io/MorphologyLoader.cpp

```diff
@@ -45,16 +48,134 @@ MorphologyLoader::MorphologyLoader(
 }
 
 #ifdef BRAYNS_USE_BRION
+bool MorphologyLoader::_importMorphologyAsMesh(
+    const servus::URI& source,
+    const size_t morphologyIndex,
+    const MaterialsMap& materials,
+    const Matrix4f& transformation,
+    TrianglesMeshMap& meshes,
+    Boxf& bounds)
+{
+    try
+    {
+        brain::neuron::Morphology morphology( source, transformation );
+        brain::neuron::SectionTypes sectionTypes;
+
+        const size_t morphologySectionTypes =
+            _geometryParameters.getMorphologySectionTypes();
+        if( morphologySectionTypes & MST_SOMA )
+            sectionTypes.push_back( brain::neuron::SectionType::soma );
+        if( morphologySectionTypes & MST_AXON )
+            sectionTypes.push_back( brain::neuron::SectionType::axon );
+        if( morphologySectionTypes & MST_DENDRITE )
+            sectionTypes.push_back( brain::neuron::SectionType::dendrite );
+        if( morphologySectionTypes & MST_APICAL_DENDRITE )
+            sectionTypes.push_back( brain::neuron::SectionType::apicalDendrite );
```

Comment:
Done, but I had to make it a free function because the return type is from brion, and since Brion is am optional dependency, that would require an #ifdef in the header.